docs/cosmology/index.rst

```diff
@@ -141,15 +147,13 @@ This can also be done more explicitly using units, which is recommended::
   >>> import astropy.units as u
   >>> cosmo = FlatLambdaCDM(H0=70 * u.km / u.s / u.Mpc, Tcmb0=2.725 * u.K, Om0=0.3)
 
-However, most of the parameters that accept units (``H0``, ``Tcmb0``)
-have default units, so unit quantities do not have to be used (with the
-exception of neutrino masses, where you must supply a unit if you want massive
-neutrinos).
+However, most of the parameters that accept units (``H0``, ``Tcmb0``) have
```

Comment:
Maybe not mention this; with the units the code is clearer...

Reply:
Removed